src/router.js

```diff
@@ -23,37 +26,54 @@ const defaultRouter = new Router({
       children: [
         {
           path: '',
+          name: 'dashboard',
+          component: DashboardPage
+        },
+        {
+          path: 'wallets',
           name: 'wallets',
-          component: WalletsPage
+          component: WalletsPage,
+          children: [
+            {
+              path: ':walletId',
+              component: Wallet
+            }
+          ]
         },
         {
           path: 'settlements',
           name: 'settlements-page',
           component: SettlementsPage,
           children: [
-            {
-              path: '',
-              name: 'settlements-waiting',
-              component: SettlementsWaiting
-            },
             {
               path: 'history',
               name: 'settlements-history',
               component: SettlementsHistory
+            },
+            {
+              path: 'incoming',
+              name: 'settlements-incoming',
+              component: SettlementsIncoming
+            },
+            {
+              path: 'outgoing',
+              name: 'settlements-outgoing',
+              component: SettlementsOutgoing
             }
           ]
         },
+        {
+          path: 'reports',
+          name: 'reports',
+          component: ReportsPage
+        },
         {
           path: 'settings',
```

Comment:
Maybe settings page needs removing because it is not linked from any page? 🤔

Reply:
We will remove it since it isn't in mockups and isn't used anymore